src/uproot/behaviors/RNTuple.py

```diff
@@ -24,6 +24,12 @@
 import uproot.source.chunk
 from uproot._util import no_filter, unset
 
+# GDS Depdencies
+try:
+    import cupy as cp
+except ImportError:
+    pass
```

Comment:
Move the import into the `_arrays_GDS` function. Then, you can catch the ImportError exception and raise a new error to the user instructing them to `python3 -m pip install uproot[GDS_cu11]` or _cu12.